docs/examples/image_binarization/image_binarization.jl

```diff
@@ -0,0 +1,128 @@
+# ---
+# title: Image Binarization
+# author: Ashwani Rathee
+# date: 2021-7-2
+# ---
+
+# ImageBinarization.jl provides several algorithms for binarizing images
+# into background and foreground(bi-level image). In this demonstration, 
+# we'll be exploring where these algorithms could be useful and learn how
+# different algorithms are suited for different types of tasks.
+
+# Suppose a person wants to make an automatic Sudoku solver using Computer 
+# Vision . Before anything else that person needs to import the image, 
+# preprocess it to optimize it for cell content extraction which can then 
+# be used for solving the sudoku. 
+
+# Let's first import packages and then testimage sudoku
+
+using Images, ImageBinarization, TestImages
+using  CoordinateTransformations, Rotations, ImageTransformations
+
+# Original Image
+
+img = testimage("sudoku")
+
+# Grayscale version as binarize! only accepts Grayscale images
+
+img = Gray.(img)
```

Comment:
I believe it is a mistake that `binarize` doesn't support `RGBA` images; it doesn't support `RGB` images. We should open an issue in ImageBinarization and provide a PR for it.

I checked the `alpha.(img)` for the sudoku image and then I realize it's all 1: `all(x->x==1, alpha.(img)) == true`. Since the alpha channel is not used at all, I think we should update our testimage and using the RGB format.

Reply:
I think we can use this as an opportunity to discuss a little bit about alpha channel. I didn't even know it was a thing, thanks to images.jl handling it all in background. Will sudoku image throw errors if sudoku is used in any other place where RGB is required?

Reply:
> Will sudoku image throw errors if sudoku is used in any other place where RGB is required?

Can you explain this a bit more? `RGBA` is not universally supported by the entire JuliaImages stack so we might just directly update our soduku image with `RGB` format in TestImages.